tls/test/test_hello_message.py

```diff
@@ -35,6 +35,13 @@ class TestClientHello(object):
         b''  # extensions.extensions
     )
 
+    supported_signature_list_extension_data = (
+        b'\x00\x0D'  # extensions[0].extension_type
+        b'\x00\x16'  # extensions[0].length
+        b'\x00\x14\x04\x01\x05\x01\x06\x01\x02\x01\x04\x03\x05\x03\x06\x03'
```

Comment:
I was rushing and made a mistake :(  Thanks for putting in the correct hashes and signatures!!